### PR TITLE
Added: Removal of RadEditorProvider references

### LINKED ISSUE #660: Telerik removal: delete RadEditorProvider from web.config
## Description of bug
I found this line in an upgraded web.config:

`<add name="HtmTemplateFileHandler" verb="*" path="*.htmtemplate" type="DotNetNuke.Providers.RadEditorProvider.HtmTemplateFileHandler, DotNetNuke.RadEditorProvider" preCondition="integratedMode" />`

## Steps to reproduce
List the steps to reproduce the behavior:
Check a web.config ;-)

## Expected result
This line should be part of the cleanup process IMO

### PR DESCRIPTION
**Pull request by Timo-Breumelhof**
fixes #660 closes #660

### CODE REVIEW

**Comment by Timo-Breumelhof** [comment]
Ehh, no idea what this error is about TBH

File: content/getting-started/setup/telerik-removal/index.md [hunk 1]
```diff
@@ -81,17 +81,21 @@ Navigate to `Settings > Extensions (Modules)` in the `Persona Bar` and uninstall
 Navigate to `Settings > Extensions (Providers)` in the `Persona Bar` and uninstall the `RadEditor Manager` extension.  Be sure to check the `Delete Files` checkbox.  NOTE: Not all installations will have this extension installed.
 
 ### Step 15
+Navigate to `Settings > Extensions (Providers)` in the `Persona Bar` and uninstall any `Telerik` related extension(s).  Be sure to check the `Delete Files` checkbox.  NOTE: Not all installations will have these extension installed.
```
Comment (david-poindexter) [inline content/getting-started/setup/telerik-removal/index.md:84]: This seems a bit vague and generalized.  We should provide specific extension name(s) here.  Thoughts?

**Review by david-poindexter — CHANGES_REQUESTED** [review]
@Timo-Breumelhof thanks so much for the PR.  See my comment and let me know your thoughts.

File: content/getting-started/setup/telerik-removal/index.md (hunk 1 above)
Comment (Timo-Breumelhof) [reply content/getting-started/setup/telerik-removal/index.md:84]: You are right it's a bit vague. My idea was that there could be other Telerik editor extensions in there, but I guess that not very likely.

**Comment by Timo-Breumelhof** [comment]
@david-poindexter change made

**Pull request merged**